selfdrive/car/honda/carstate.py

```diff
@@ -277,20 +254,15 @@ def update(self, cp, cp_cam, cp_body):
     gear = int(cp.vl[self.gearbox_msg]["GEAR_SHIFTER"])
     ret.gearShifter = self.parse_gear_shifter(self.shifter_values.get(gear, None))
 
-    pedal_gas = cp.vl["POWERTRAIN_DATA"]["PEDAL_GAS"]
-    # crv doesn't include cruise control
-    if self.CP.carFingerprint in (CAR.CRV, CAR.CRV_EU, CAR.HRV, CAR.ODYSSEY, CAR.ACURA_RDX, CAR.RIDGELINE, CAR.PILOT_2019, CAR.ODYSSEY_CHN):
-      ret.gas = pedal_gas / 256.
-    else:
-      ret.gas = cp.vl["GAS_PEDAL_2"]["CAR_GAS"] / 256.
+    ret.gas = cp.vl["POWERTRAIN_DATA"]["PEDAL_GAS"]
 
     # this is a hack for the interceptor. This is now only used in the simulation
     # TODO: Replace tests by toyota so this can go away
     if self.CP.enableGasInterceptor:
       user_gas = (cp.vl["GAS_SENSOR"]["INTERCEPTOR_GAS"] + cp.vl["GAS_SENSOR"]["INTERCEPTOR_GAS2"]) / 2.
-      ret.gasPressed = user_gas > 1e-5  # this works because interceptor read < 0 when pedal position is 0. Once calibrated, this will change
+      ret.gasPressed = user_gas > 1e-5  # this works because interceptor reads < 0 when pedal position is 0. Once calibrated, this will change
     else:
-      ret.gasPressed = pedal_gas > 1e-5
+      ret.gasPressed = ret.gas > 1e-5
```

Comment:
This refactor isn't equivalent, since `ret.gas` is scaled by 1/256.  Is that OK?

Reply:
yeah, that should be fine